test/Faker/Provider/fr_CH/PersonTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Faker\Test\Provider\fr_CH;
+
+use Faker\Generator;
+use Faker\Provider\fr_CH\Person;
+use PHPUnit\Framework\TestCase;
+
+class PersonTest extends TestCase
+{
+    private $faker;
+
+    public function setUp()
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Person($faker));
+        $this->faker = $faker;
+    }
+
+	public function testAvsNumber()
+    {
+		$avs = $this->faker->navs13;
+		$this->assertRegExp('/^756\.([0-9]{4})\.([0-9]{4})\.([0-9]{2})$/i', $avs);
```

Comment:
Does it make sense to do an assertion about the checksum here? Would that suffice to check the whole number?

Reply:
Indeed, a checksum verification should be enough to assert the number is valid.

Reply:
Actually, after a bit of thinking, I believe it's still pertinent to assert the format of the generated number since the format is part of the specification. Sure, the checksum would confirm that the number is a valid EAN-13 number but it could validate any EAN-13 that is NOT an AVS13 number.